launch/tier4_perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml

```diff
@@ -11,6 +11,13 @@
   <arg name="container_name" default="pointcloud_container"/>
   <arg name="use_validator" default="true" description="use obstacle_pointcloud based validator"/>
   <arg name="score_threshold" default="0.35"/>
+  <arg name="centerpoint_model_name" default="centerpoint_tiny"/>
+  <arg name="centerpoint_model_path" default="$(find-pkg-share lidar_centerpoint)/data"/>
+  <arg name="lidar_model_param_path" default="$(find-pkg-share lidar_centerpoint)/config"/>
+  <arg name="objects_validation_method" default="obstacle_pointcloud"/>
+  <arg name="objects_filter_method" default="position_filter"/>
```

Comment:
[Q] Could you change default objects_filter_method to "lanelet_filter" ?

Reply:
@miursh Done.